src/arrayMethodSort.js

```diff
@@ -5,8 +5,76 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction =
    (a, b) => String(a) > String(b) ? 1 : -1) {
```
You can simplify your code